packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -87,9 +87,10 @@ export abstract class ImportBase {
 
           // java!
           if (options.targetLanguage === Language.JAVA) {
+            const javaName = name.replace('/', '.')
```

Comment:
I never remember if this replaces all occurrences or just the first. I always use a reflex `/\//g`

Reply:
Will do that!